### ISSUE TITLE
OOM killer triggered by an user space application put the master node (and cluster) to unreachable state

### ISSUE THREAD

**Issue by oplatek** [aws/aws-parallelcluster#2701]
Hi there!

Short description of a situation:
1. Several users were using a cluster and a badly configured script on the head node triggered OOM killer. 
  - While the script was still running the head node was not reachable by ssh -- all terminals froze.
2. After the script was interrupted, the cluster was usable for a short time but then the master node became unreachable again.
3. I restarted the master instance and copied out the whole `/var/log` folder.

I pasted below the `/var/log/syslog` from the relevant time.
It seems that there was an FSX problem.

**My questions**
- How can we prevent becoming cluster unstable except by instructing the users not to run tasks on a head node?
- Can you give me hints what was the root cause of the head node unreachability?
- How can I recover the parallel cluster state if compute nodes are "fine" and the master node is unreachable?


**Required Info:**
 - AWS ParallelCluster version 2.10.3
 - See `/var/log/syslog` below
```
Apr 30 13:17:48 ip-10-0-33-4 kernel: [266333.003777] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/user.slice,task=python,pid=15850,uid=1000                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:17:48 ip-10-0-33-4 kernel: [266333.003891] Out of memory: Killed process 15850 (python) total-vm:11510320kB, anon-rss:9087884kB, file-rss:0kB, shmem-rss:0kB, UID:1000 pgtables:18184kB oom_score_adj:0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:17:49 ip-10-0-33-4 systemd-networkd[714]: ens5: Could not set DHCPv4 route: Connection timed out                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Apr 30 13:17:49 ip-10-0-33-4 kernel: [266333.297289] oom_reaper: reaped process 15850 (python), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Watchdog timeout (limit 5min)!                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 13:17:49 ip-10-0-33-4 snapd[16053]: AppArmor status: apparmor is enabled and all features are available                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Killing process 2545 (snapd) with signal SIGABRT.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16899 '/kernel/slab/radix_tree_node/cgroup/radix_tree_node(5102:cups-browsed.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16899 '/kernel/slab/radix_tree_node/cgroup/radix_tree_node(5102:cups-browsed.service)' killed                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [15995] terminated by signal 9 (KILL)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [15995] failed while handling '/kernel/slab/radix_tree_node/cgroup/radix_tree_node(5102:cups-browsed.service)'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 13:17:49 ip-10-0-33-4 CRON[16079]: (root) CMD (  [ -x /usr/lib/php/sessionclean ] && if [ ! -d /run/systemd/system ]; then /usr/lib/php/sessionclean; fi)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: State 'stop-sigabrt' timed out. Terminating.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16900 '/kernel/slab/kmalloc-96/cgroup/kmalloc-96(2136:munge.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16902 '/kernel/slab/kmalloc-4k/cgroup/kmalloc-4k(2136:munge.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16902 '/kernel/slab/kmalloc-4k/cgroup/kmalloc-4k(2136:munge.service)' killed                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [16012] terminated by signal 9 (KILL)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [16012] failed while handling '/kernel/slab/kmalloc-4k/cgroup/kmalloc-4k(2136:munge.service)'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16903 '/kernel/slab/kmalloc-8k/cgroup/kmalloc-8k(2136:munge.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 CRON[16084]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16903 '/kernel/slab/kmalloc-8k/cgroup/kmalloc-8k(2136:munge.service)' killed                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:17:49 ip-10-0-33-4 anacron[16096]: Anacron 2.3 started on 2021-04-30                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [16022] terminated by signal 9 (KILL)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: worker [16022] failed while handling '/kernel/slab/kmalloc-8k/cgroup/kmalloc-8k(2136:munge.service)'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: State 'stop-sigterm' timed out. Killing.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 anacron[16096]: Normal exit (0 jobs run)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Killing process 2545 (snapd) with signal SIGKILL.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Main process exited, code=killed, status=9/KILL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Failed with result 'watchdog'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16904 '/kernel/slab/ext4_inode_cache/cgroup/ext4_inode_cache(1865:snapd.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:17:49 ip-10-0-33-4 systemd-udevd[494]: seq 16906 '/kernel/slab/TCP/cgroup/TCP(1865:snapd.service)' is taking a long time                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Service hold-off time over, scheduling restart.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: snapd.service: Scheduled restart job, restart counter is at 2.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: Stopped Snap Daemon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: Starting Snap Daemon...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: Starting Clean php session files...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:17:49 ip-10-0-33-4 systemd[1]: Started Run anacron jobs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:17:50 ip-10-0-33-4 systemd[1]: Started Clean php session files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:17:50 ip-10-0-33-4 snapd[16053]: AppArmor status: apparmor is enabled and all features are available                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:17:50 ip-10-0-33-4 snapd[16053]: daemon.go:347: started snapd/2.49.2 (series 16; classic) ubuntu/18.04 (amd64) linux/5.4.0-1039-aws.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:17:50 ip-10-0-33-4 snapd[16053]: daemon.go:440: adjusting startup timeout by 45s (pessimistic estimate of 30s plus 5s per snap)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:17:50 ip-10-0-33-4 systemd[1]: Started Snap Daemon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 13:20:05 ip-10-0-33-4 systemd[1]: Started Session 296 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:20:26 ip-10-0-33-4 systemd[1]: serial-getty@ttyS0.service: Service hold-off time over, scheduling restart.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:20:26 ip-10-0-33-4 systemd[1]: serial-getty@ttyS0.service: Scheduled restart job, restart counter is at 1.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:20:26 ip-10-0-33-4 systemd[1]: Stopped Serial Getty on ttyS0.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:20:26 ip-10-0-33-4 systemd[1]: Started Serial Getty on ttyS0.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:20:27 ip-10-0-33-4 systemd[1]: Started Session 297 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:20:41 ip-10-0-33-4 systemd[1]: Started Session 298 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:23:14 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:23:14 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:25:39 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:25:39 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:26:32 ip-10-0-33-4 pcluster_ssh_target_checker: Target Host github.com is not in any VPC CIDR 10.0.0.0/16                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:26:55 ip-10-0-33-4 pcluster_ssh_target_checker: Target Host github.com is not in any VPC CIDR 10.0.0.0/16                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:27:41 ip-10-0-33-4 systemd[1]: Stopping Slurm controller daemon...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:27:41 ip-10-0-33-4 systemd[1]: Stopped Slurm controller daemon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:27:41 ip-10-0-33-4 systemd[1]: Starting Slurm controller daemon...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:27:42 ip-10-0-33-4 systemd[1]: slurmctld.service: Can't open PID file /var/run/slurmctld.pid (yet?) after start: No such file or directory                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:27:42 ip-10-0-33-4 systemd[1]: Started Slurm controller daemon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:28:18 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:28:18 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:36:57 ip-10-0-33-4 pcluster_ssh_target_checker: Target Host c5n-nxlarge-dy-c5n9xlarge-1 is in VPC CIDR 10.0.0.0/16                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:39:00 ip-10-0-33-4 systemd[1]: Starting Clean php session files...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:39:00 ip-10-0-33-4 systemd[1]: Started Clean php session files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:39:01 ip-10-0-33-4 CRON[19238]: (root) CMD (  [ -x /usr/lib/php/sessionclean ] && if [ ! -d /run/systemd/system ]; then /usr/lib/php/sessionclean; fi)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:39:49 ip-10-0-33-4 systemd[1]: Started Session 300 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 13:46:56 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:46:56 ip-10-0-33-4 systemd-resolved[2253]: Server returned error NXDOMAIN, mitigating potential DNS violation DVE-2018-0001, retrying transaction with reduced feature level UDP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:49:48 ip-10-0-33-4 avahi-daemon[888]: Withdrawing address record for 10.0.33.4 on ens5.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:49:48 ip-10-0-33-4 avahi-daemon[888]: Leaving mDNS multicast group on interface ens5.IPv4 with address 10.0.33.4.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 13:49:48 ip-10-0-33-4 avahi-daemon[888]: Interface ens5.IPv4 no longer relevant for mDNS.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:49:48 ip-10-0-33-4 whoopsie[1385]: [13:49:48] Cannot reach: https://daisy.ubuntu.com                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:50:03 ip-10-0-33-4 kernel: [268267.304574] Lustre: 3317:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1619790596/real 1619790596]  req@0000000020cf8873 x1698193162592128/t0(0) o4->no2zxbmv-OST0000-osc-000000001785e248@10.0.46.139@tcp:6/4 lens 608/448 e 0 to 1 dl 1619790603 ref 2 fl Rpc:X/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:50:03 ip-10-0-33-4 kernel: [268267.304585] Lustre: no2zxbmv-OST0000-osc-000000001785e248: Connection to no2zxbmv-OST0000 (at 10.0.46.139@tcp) was lost; in progress operations using this service will wait for recovery to complete                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:50:09 ip-10-0-33-4 kernel: [268273.448569] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1619790603/real 1619790603]  req@0000000031d847bc x1698193162592160/t0(0) o8->no2zxbmv-OST0000-osc-000000001785e248@10.0.46.139@tcp:28/4 lens 520/544 e 0 to 1 dl 1619790609 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:50:10 ip-10-0-33-4 kernel: [268274.472572] Lustre: 3318:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1619790603/real 1619790603]  req@00000000fa2d1c62 x1698193162592144/t0(0) o400->no2zxbmv-MDT0000-mdc-000000001785e248@10.0.47.232@tcp:12/10 lens 224/224 e 0 to 1 dl 1619790610 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 13:50:10 ip-10-0-33-4 kernel: [268274.472582] Lustre: no2zxbmv-MDT0000-mdc-000000001785e248: Connection to no2zxbmv-MDT0000 (at 10.0.47.232@tcp) was lost; in progress operations using this service will wait for recovery to complete                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 13:50:16 ip-10-0-33-4 kernel: [268280.616562] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1619790610/real 1619790610]  req@0000000052a0ed37 x1698193162592192/t0(0) o38->no2zxbmv-MDT0000-mdc-000000001785e248@10.0.47.232@tcp:12/10 lens 520/544 e 0 to 1 dl 1619790616 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 13:50:17 ip-10-0-33-4 kernel: [268281.640569] LustreError: 166-1: MGC10.0.47.232@tcp: Connection to MGS (at 10.0.47.232@tcp) was lost; in progress operations using this service will fail                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 13:50:23 ip-10-0-33-4 kernel: [268287.784552] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1619790617/real 1619790617]  req@000000004e75e2a6 x1698193162592224/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619790623 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:50:23 ip-10-0-33-4 kernel: [268287.784555] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:50:43 ip-10-0-33-4 kernel: [268307.496583] LNetError: 3306:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:50:43 ip-10-0-33-4 kernel: [268307.504218] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:50:43 ip-10-0-33-4 kernel: [268307.512121] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619790643/real 1619790643]  req@000000004e75e2a6 x1698193162592240/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619790654 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.120538] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.120562] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.128474] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.128508] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619790669/real 1619790669]  req@0000000055a460e3 x1698193162592304/t0(0) o38->no2zxbmv-MDT0000-mdc-000000001785e248@10.0.47.232@tcp:12/10 lens 520/544 e 0 to 1 dl 1619790680 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.136556] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:51:09 ip-10-0-33-4 kernel: [268334.148415] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 2 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:51:36 ip-10-0-33-4 kernel: [268360.744510] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:51:36 ip-10-0-33-4 kernel: [268360.744534] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:51:36 ip-10-0-33-4 kernel: [268360.752284] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:51:36 ip-10-0-33-4 kernel: [268360.767907] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.368472] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.368492] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.376240] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.376273] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619790723/real 1619790723]  req@00000000d89d2005 x1698193162592400/t0(0) o38->no2zxbmv-MDT0000-mdc-000000001785e248@10.0.47.232@tcp:12/10 lens 520/544 e 0 to 1 dl 1619790739 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.384338] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:52:03 ip-10-0-33-4 kernel: [268387.396198] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 5 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:52:29 ip-10-0-33-4 kernel: [268413.992426] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:52:29 ip-10-0-33-4 kernel: [268413.992446] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:52:29 ip-10-0-33-4 kernel: [268414.000287] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:52:29 ip-10-0-33-4 kernel: [268414.015921] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:52:56 ip-10-0-33-4 kernel: [268440.616386] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:52:56 ip-10-0-33-4 kernel: [268440.616405] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:52:56 ip-10-0-33-4 kernel: [268440.624015] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:53:23 ip-10-0-33-4 kernel: [268467.240343] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619790803/real 1619790803]  req@00000000c350db1d x1698193162592528/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619790844 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 13:53:23 ip-10-0-33-4 kernel: [268467.240345] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 8 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:53:23 ip-10-0-33-4 kernel: [268467.240390] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:53:23 ip-10-0-33-4 kernel: [268467.248267] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:53:23 ip-10-0-33-4 kernel: [268467.256299] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:54:16 ip-10-0-33-4 kernel: [268520.488293] LNetError: 3304:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:54:16 ip-10-0-33-4 kernel: [268520.496035] LNetError: 3304:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:54:16 ip-10-0-33-4 kernel: [268520.503583] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:54:16 ip-10-0-33-4 kernel: [268520.511514] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.360157] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619790936/real 1619790936]  req@00000000ab6a7c9c x1698193162592800/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619790991 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.360160] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 14 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.360205] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.367904] LNetError: 3305:0:(lib-socket.c:486:lnet_sock_create()) Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.375426] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:55:36 ip-10-0-33-4 kernel: [268600.383367] LNetError: Skipped 1 previous similar message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 13:57:49 ip-10-0-33-4 kernel: [268733.479945] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 13:57:49 ip-10-0-33-4 kernel: [268733.487694] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 2 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 13:57:49 ip-10-0-33-4 kernel: [268733.495262] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 13:57:49 ip-10-0-33-4 kernel: [268733.503135] LNetError: Skipped 2 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:00:02 ip-10-0-33-4 kernel: [268866.599744] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619791202/real 1619791202]  req@00000000e1b40888 x1698193162593296/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619791257 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:00:02 ip-10-0-33-4 kernel: [268866.599746] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 29 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:02:15 ip-10-0-33-4 kernel: [268999.719588] LNetError: 3304:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:02:15 ip-10-0-33-4 kernel: [268999.727298] LNetError: 3304:0:(lib-socket.c:486:lnet_sock_create()) Skipped 6 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 14:02:15 ip-10-0-33-4 kernel: [268999.734825] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.46.139@tcp at host 10.0.46.139 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:02:15 ip-10-0-33-4 kernel: [268999.742761] LNetError: Skipped 6 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:03:52 ip-10-0-33-4 systemd[1]: Started Run anacron jobs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 14:03:52 ip-10-0-33-4 anacron[20936]: Anacron 2.3 started on 2021-04-30                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 14:03:52 ip-10-0-33-4 anacron[20936]: Normal exit (0 jobs run)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 14:08:54 ip-10-0-33-4 kernel: [269399.078938] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619791734/real 1619791734]  req@000000005913ab84 x1698193162594448/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619791789 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:08:54 ip-10-0-33-4 kernel: [269399.078941] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 59 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:09:01 ip-10-0-33-4 CRON[21012]: (root) CMD (  [ -x /usr/lib/php/sessionclean ] && if [ ! -d /run/systemd/system ]; then /usr/lib/php/sessionclean; fi)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 14:09:08 ip-10-0-33-4 systemd[1]: Starting Clean php session files...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 14:09:09 ip-10-0-33-4 systemd[1]: Started Clean php session files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:11:08 ip-10-0-33-4 kernel: [269532.198736] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:11:08 ip-10-0-33-4 kernel: [269532.206713] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 12 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 14:11:08 ip-10-0-33-4 kernel: [269532.214538] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:11:08 ip-10-0-33-4 kernel: [269532.222568] LNetError: Skipped 12 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:17:01 ip-10-0-33-4 CRON[21156]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 14:19:07 ip-10-0-33-4 kernel: [270011.430030] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619792347/real 1619792347]  req@000000001fed23a5 x1698193162595712/t0(0) o8->no2zxbmv-OST0000-osc-000000001785e248@10.0.46.139@tcp:28/4 lens 520/544 e 0 to 1 dl 1619792402 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:19:07 ip-10-0-33-4 kernel: [270011.430033] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 68 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:21:47 ip-10-0-33-4 kernel: [270171.173788] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:21:47 ip-10-0-33-4 kernel: [270171.181596] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 14:21:47 ip-10-0-33-4 kernel: [270171.189277] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:21:47 ip-10-0-33-4 kernel: [270171.197206] LNetError: Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:29:19 ip-10-0-33-4 kernel: [270623.781103] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619792959/real 1619792959]  req@000000004e75e2a6 x1698193162603984/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619793014 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:29:19 ip-10-0-33-4 kernel: [270623.781105] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 68 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:32:25 ip-10-0-33-4 kernel: [270810.148833] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:32:25 ip-10-0-33-4 kernel: [270810.156677] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 14:32:25 ip-10-0-33-4 kernel: [270810.164400] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:32:26 ip-10-0-33-4 kernel: [270810.287622] LNetError: Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:39:01 ip-10-0-33-4 CRON[21355]: (root) CMD (  [ -x /usr/lib/php/sessionclean ] && if [ ! -d /run/systemd/system ]; then /usr/lib/php/sessionclean; fi)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 14:39:19 ip-10-0-33-4 systemd[1]: Starting Clean php session files...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 14:39:19 ip-10-0-33-4 systemd[1]: Started Clean php session files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:39:31 ip-10-0-33-4 kernel: [271236.132175] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619793571/real 1619793571]  req@00000000741583b3 x1698193162605568/t0(0) o250->MGC10.0.47.232@tcp@10.0.47.232@tcp:26/25 lens 520/544 e 0 to 1 dl 1619793626 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:39:31 ip-10-0-33-4 kernel: [271236.132178] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 68 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:43:04 ip-10-0-33-4 kernel: [271449.123875] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:43:04 ip-10-0-33-4 kernel: [271449.131854] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 14:43:04 ip-10-0-33-4 kernel: [271449.139702] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:43:04 ip-10-0-33-4 kernel: [271449.147824] LNetError: Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:49:44 ip-10-0-33-4 kernel: [271848.483268] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1619794184/real 1619794184]  req@000000004161d92d x1698193162607248/t0(0) o8->no2zxbmv-OST0000-osc-000000001785e248@10.0.46.139@tcp:28/4 lens 520/544 e 0 to 1 dl 1619794239 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:49:44 ip-10-0-33-4 kernel: [271848.483270] Lustre: 3315:0:(client.c:2116:ptlrpc_expire_one_request()) Skipped 68 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:53:43 ip-10-0-33-4 kernel: [272088.098901] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Error trying to bind to port 1023: -99                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:53:43 ip-10-0-33-4 kernel: [272088.106781] LNetError: 3308:0:(lib-socket.c:486:lnet_sock_create()) Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 14:53:43 ip-10-0-33-4 kernel: [272088.114505] LNetError: 11e-e: Unexpected error -99 connecting to 10.0.47.232@tcp at host 10.0.47.232 on port 988                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:53:43 ip-10-0-33-4 kernel: [272088.122542] LNetError: Skipped 15 previous similar messages                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: Stopped Unattended Upgrades Shutdown.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-279.scope: Killing process 4118 (mosh-server) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-279.scope: Killing process 4119 (tmux-change-hom) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-279.scope: Killing process 4120 (tmux: client) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: Stopping Session 279 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-297.scope: Killing process 17141 (mosh-server) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-297.scope: Killing process 17142 (tmux-change-hom) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-297.scope: Killing process 17143 (tmux: client) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: Stopping Session 297 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: session-110.scope: Killing process 13056 (ssh-agent) with signal SIGTERM.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: Stopping Session 110 of user ubuntu.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 14:53:57 ip-10-0-33-4 amazon-ssm-agent.amazon-ssm-agent[1167]: 2021-04-30 14:53:57 INFO [amazon-ssm-agent] Got signal:terminated value:0x55f514f346e0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 14:53:57 ip-10-0-33-4 amazon-ssm-agent.amazon-ssm-agent[1167]: 2021-04-30 14:53:57 INFO [amazon-ssm-agent] Stopping Core Agent                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 14:53:57 ip-10-0-33-4 amazon-ssm-agent.amazon-ssm-agent[1167]: 2021-04-30 14:53:57 INFO [amazon-ssm-agent] [LongRunningWorkerContainer] Receiving stop signal, stop worker monitor                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 14:53:57 ip-10-0-33-4 ModemManager[980]: <info>  Caught signal, shutting down...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 14:53:57 ip-10-0-33-4 systemd[1]: Stopping User Manager for UID 1000...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
Apr 30 14:53:57 ip-10-0-33-4 snapd[16053]: main.go:155: Exiting on terminated signal.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'lp'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'ppdev'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'parport_pc'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'ib_core'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'ib_uverbs'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'efa'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Failed to find module 'lnet'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 15:00:48 ip-10-0-33-4 blkmapd[473]: open pipe file /run/rpc_pipefs/nfs/blocklayout failed: No such file or directory                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounted Huge Pages File System.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Started pNFS block layout mapping daemon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Started Load/Save Random Seed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Flush Journal to Persistent Storage...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'iscsi_tcp'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Started udev Coldplug all Devices.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Started Monitoring of LVM2 mirrors, snapshots etc. using dmeventd or progress polling.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Started Create Static Device Nodes in /dev.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting udev Kernel Device Manager...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Reached target Local File Systems (Pre).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounting Mount unit for core18, revision 1997...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounting Mount unit for snapd, revision 11588...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 15:00:48 ip-10-0-33-4 systemd-modules-load[439]: Inserted module 'ib_iser'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounting Mount unit for snapd, revision 11107...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounting Mount unit for amazon-ssm-agent, revision 3552...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Mounting Mount unit for core18, revision 1988...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Reached target Local File Systems.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting AppArmor initialization...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Enable support for additional executable binary formats...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Set console font and keymap...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Clean up any mess left by 0dns-up...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Tell Plymouth To Write Out Runtime Data...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting Preprocess NFS configuration...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Starting ebtables ruleset management...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: systemd-modules-load.service: Main process exited, code=exited, status=1/FAILURE                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: systemd-modules-load.service: Failed with result 'exit-code'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
Apr 30 15:00:48 ip-10-0-33-4 systemd[1]: Failed to start Load Kernel Modules.                                            
```

**Comment by yuleiwan** [comment]
Hi @optalek,
Corresponding to your questions:

* How can we prevent becoming cluster unstable except by instructing the users not to run tasks on a head node?
In current pcluster version, we are unable to prevent the user to do operations on head node. We are planning to introduce multi-user environment in next major version, that will prevent operations from users who are not owner of cluster. 

* Can you give me hints what was the root cause of the head node unreachability?
I think because the ssh daemon running on the server would not have enough resources available to allow it to start a new process for an incoming ssh session due to oom.

* How can I recover the parallel cluster state if compute nodes are "fine" and the master node is unreachable?
If you want to recover the the master node, compute nodes should be stopped, otherwise, we can't replace the volume.

From `/var/log/syslog`, it looks like there is no space for starting a process and it took long time for requesting a process.

Thanks,
Yulei

**Comment by oplatek** [comment]
Hi @yuleiwan ! Thank you for a prompt reply!

---

> In current pcluster version, we are unable to prevent the user to do operations on head node. We are planning to introduce multi-user environment in next major version, that will prevent operations from users who are not owner of cluster.

Awesome! Looking forward to mult-user environment in the next version.


---


> How can I recover the parallel cluster state if compute nodes are "fine" and the master node is unreachable?
If you want to recover the master node, compute nodes should be stopped, otherwise, we can't replace the volume.

What do you mean exactly by recover the master node here? 
1. Just keep the master node running and run `pcluster stop $clustername`, and wait?
2. Do something else with master node? (I tried stop & start the master node from AWS console (browser)).  And than run `pcluster stop $clustername`?

**Comment by yuleiwan** [comment]
Hi oplatek,
I think you should stop the compute fleet at first if `pcluster stop $clustername` command still works, then check which volume is out of memory, you should create a new volume from snapshot with enough space and replace that volume on head node(This step may need stop the head node).

**Comment by oplatek** [comment]
> We are planning to introduce multi-user environment in next major version, that will prevent operations from users who are not owner of cluster.

When it will roughly be? Several months, years? (Do you mean 2.11.? or 3.?.? major release)

**Comment by yuleiwan** [comment]
it won’t be included until the next major version, but that it’s not clear when exactly

**Comment by rexcsn** [comment]
Hi @oplatek ,

Wanted to clarify on some previous comments. The multi-user feature is still under planning, so its scope will likely change. However, it is unlikely that this feature will prevent the OOM issues you experienced.
Unfortunately, we do not have specific suggestion on how to prevent OOM issues other than typical Linux practices to prevent OOM.

For general information on how to recover from OOM situation for EC2 instances please see [this troubleshooting guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/TroubleshootingInstances.html#MemoryOOM), which suggests rebooting the instance and changing to a larger instance type with more memory.
Rebooting or replacing the head node is not officially supported by ParallelCluster due to issues with some edge cases, but in most cases it should work and the cluster should still be operational after the head node reboot.
As for using larger instance type, looks like running jobs on the head node is not your typical use case, so this suggestion might be less applicable.

The comments @yuleiwan mentioned about replacing the volume would be applicable if you were out of storage space on an EBS volume, or another shared storage, which is different from the OOM issue you experienced.

Finally, wanted to double check and make sure you are unblocked.
Were you able to restore the cluster to working condition after restarting/rebooting the head node?
Please let us know if you are experiencing an issue or have open question.

Thank you!

**Comment by oplatek** [comment]
Hi @rexcsn,

oh, I understood the previous answer that I ran out of swap space on the EBS storage after running out of OOM.
Thank you for your clarification.

Yes, I am not blocked by this. I managed to backup FSX to S3 and recreated the cluster from scratch.

**Comment by rexcsn** [comment]
Since there is no open question I will close the issue. Please feel free to re-open if there is any question.

Thank you!
